### ISSUE TITLE
Rails server crashes on rake db:nuke_pave or on server query

### ISSUE THREAD

**Issue by ioncrash** [ga-wdi-boston/capstone-project#485]
running rake db:nuke_pave now gives me the following error:

```
could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
```

brew services list gives me this:
```
mongodb    started pygartheangel /Users/pygartheangel/Library/LaunchAgents/homebrew.mxcl.mongodb.plist
postgresql started pygartheangel /Users/pygartheangel/Library/LaunchAgents/homebrew.mxcl.postgresql.plist
```

brew services restart postgresql gives me:
```
==> Successfully stopped `postgresql` (label: homebrew.mxcl.postgresql)
==> Successfully started `postgresql` (label: homebrew.mxcl.postgresql)
```

tail /usr/local/var/log/postgres.log gives me:

```
FATAL:  database files are incompatible with server
DETAIL:  The data directory was initialized by PostgreSQL version 9.5, which is not compatible with this version 9.6.1.
FATAL:  database files are incompatible with server
DETAIL:  The data directory was initialized by PostgreSQL version 9.5, which is not compatible with this version 9.6.1.
FATAL:  database files are incompatible with server
DETAIL:  The data directory was initialized by PostgreSQL version 9.5, which is not compatible with this version 9.6.1.
FATAL:  database files are incompatible with server
DETAIL:  The data directory was initialized by PostgreSQL version 9.5, which is not compatible with this version 9.6.1.
FATAL:  database files are incompatible with server
DETAIL:  The data directory was initialized by PostgreSQL version 9.5, which is not compatible with this version 9.6.1.
```

I also tried rm /usr/local/var/postgres/postmaster.pid
which gave me

`rm: /usr/local/var/postgres/postmaster.pid: No such file or directory`

Thank you!

**Comment by gaand** [comment]
https://github.com/ga-wdi-boston/capstone-project/issues?q=is%3Aissue+postgres+is%3Aclosed

**Comment by ioncrash** [comment]
Ah, ok I thought I had tried all of the closed issues but apparently missed #479 which resolved my issue. Sorry and thanks

**Comment by gaand** [comment]
@ioncrash No need to apologize.  I'm glad you got it fixed.
